src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static lotto.utils.Constant.LOTTO_PRICE;
+
+public class LottoResult {
+    private Map<Rank, Integer> result = new HashMap<>();
+
+    public LottoResult(Map<Rank, Integer> result) {
+        this.result = result;
+    }
+
+    public Map<Rank, Integer> getResult() {
+        return result;
+    }
+
+    public double calculateProfitRate() {
+        int totalPrize = calculateTotalPrize();
+        int totalExpense = result.values().stream().mapToInt(count -> count).sum() * LOTTO_PRICE;
```

Comment:
개행 처리 놓치신거 같아 코멘트 드립니다!
```suggestion
        int totalExpense = result.values()
        .stream()
        .mapToInt(count -> count)
        .sum() * LOTTO_PRICE;
```